slidingwindow.go

```diff
@@ -136,7 +136,7 @@ func (s *SlidingWindowRedis) Increment(ctx context.Context, prev, curr time.Time
 	case <-done:
 		if err == redis.TxFailedErr {
 			return 0, 0, errors.Wrap(err, "redis transaction failed")
-		} else if err != nil && err.Error() == "redis: nil" { // TODO: is there an exported error of that type?
+		} else if err != nil && err == redis.Nil {
```

Comment:
I know this was there before but is the check for `err != nill` necessary here? Can we just do this?

```golang
} else if err == redis.Nil {
```

Reply:
I fixed this, but I have to check the tests to see if I can do what you asked regarding a test case to cover this branch

Reply:
Ok, let me know if you are willing to add such a test, otherwise i will merge as is and add a test later

Reply:
merge this as it is, please